post_office/templatetags/html_email.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from email.mime.image import MIMEImage
+import hashlib
+import os
+from django import template
+from django.conf import settings
+from django.contrib.staticfiles import finders
+from django.core.files import File
+from django.core.files.images import ImageFile
+
+register = template.Library()
+
+
+@register.simple_tag(takes_context=True)
+def image_src(context, file):
```

Comment:
Can we please rename this to `inlined_image`?  I also wonder if we can turn this into a template filter instead. So it can be used as such `<img src = "{{ 'foo.png'|inlined }}">`

Reply:
This is the counterpart of Django's templatetag `static`, therefore I would stick with that convention, rather than using a filter. By using `image_src` I want to emphasize that it is a kind of source object. I also used this naming convention [here](https://github.com/jrief/django-sass-processor/blob/0503663de4ff50e2858bb5e052f0fa96aa1f9003/sass_processor/templatetags/sass_tags.py#L48). 
But I've no objections against renaming it to `inline_image`.

Reply:
Ok, let's use `inline_image` template tag then :)